ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -139,6 +158,15 @@ public void shouldLoadUdfs() {
     verify(udfLoader).load();
   }
 
+  @Test
+  public void shouldCreateProcessingLogTopic() {
```

Comment:
Be good to have another test that proves the topic is _not_ created if the config doesn't ask for it.